librabbitmq/amqp_openssl.c

```diff
@@ -583,8 +583,8 @@ void amqp_set_initialize_ssl_library(amqp_boolean_t do_initialize) {
   CHECK_SUCCESS(pthread_mutex_unlock(&openssl_init_mutex));
 }
 
-static unsigned long ssl_threadid_callback(void) {
-  return (unsigned long)pthread_self();
+static pthread_t ssl_threadid_callback(void) {
```

Comment:
The clang-format check is failing on this, please run clang-format on this changed file.